io_mesh_w3d/export_utils_w3d.py

```diff
@@ -263,7 +268,9 @@ def append_hlod_subObject(hlod, container_name, mesh_struct, hierarchy):
 
     if not mesh_struct.is_skin():
         for index, pivot in enumerate(hierarchy.pivots):
-            if pivot.name == name:
+            pivot_name = pivot.name.replace("B_", "").replace("BAT_","")
```

Comment:
I think this is only be convention. We shouldn't rely on that pattern.

Reply:
I dont see any other way to determine which pivto is a bone and which pivot is not in these specific cases. So unless we dont have any, we have to rely on that.